redux-router/package.json

```diff
@@ -1,5 +1,6 @@
 {
   "dependencies": {
+    "@types/react-router": "^2.0.41",
```

Comment:
I'll be working on supporting multiple versions of a package (Microsoft/types-publisher#3), so maybe this pull request could wait until then?